scripts/utils.js

```diff
@@ -0,0 +1,82 @@
+export function getTotalHeight(element) {
+  const style = getComputedStyle(element);
+  return (
+    element.offsetHeight +
+    parseInt(style.marginTop) +
+    parseInt(style.marginBottom)
+  );
+}
+
+export function getQuestionHeight(question) {
+  const h3 = question.querySelector('h3');
+  const h3Height = getTotalHeight(h3);
+  const questionStyle = getComputedStyle(question);
+  return (
+    h3Height +
+    parseInt(questionStyle.paddingTop) +
+    parseInt(questionStyle.paddingBottom)
+  );
+}
+
+export function loadQuestions(elements) {
+  const openQuestion = elements.questionSection.querySelector('.open-question');
+  const openAnswer = openQuestion.querySelector('.question__answer');
+  const openAnswerHeight = getTotalHeight(openAnswer);
+  const questionHeight = getQuestionHeight(openQuestion);
+  openQuestion.style['max-height'] = `${questionHeight + openAnswerHeight}px`;
```

Comment:
avoid inline styles if possible

Reply:
I do need to access this property because I am calculating the max-height dynamically, so I can't have a class with that property in advance, or at least is easier this way.

Reply:
is it possible to use css classes instead?

Reply:
I guess I could create the class dynamically with js, but I don't sure if it is a better idea. And I need the dynamic height for this implementation